public/pages/SampleData/utils/constants.tsx

```diff
@@ -183,19 +183,27 @@ export const sampleEcommerce = {
         },
       },
       {
-        feature_name: 'sum_total_revenue',
+        feature_name: 'avg_total_revenue',
```

Comment:
So this sample detector has 4 features: `sum_failed_items`, `avg_total_revenue`, `max_total_revenue`, and `min_total_revenue`. What story we want to tell user with this sample detector?

Reply:
My thoughts on adding the max() and min() revenue here was to pick up on anomalous spikes related to single transactions made. The sample data can have anomalous increase spikes if an expensive transaction is made, and can have anomalous decrease spikes (such as 0) if there was failed transactions. Since there can be 10 or more transactions made per interval, the avg() isn't always best at picking this up.

Overall, the story I want to tell with this detector is find any anomalous points where extremely large purchases were made, or when there were any failed transactions.